tsconfig.json

```diff
@@ -15,7 +14,7 @@
     "target": "es2016"
   },
   "include": ["./src", "typings/*.d.ts"],
-  "exclude": ["node_modules/*"],
+  "exclude": ["node_modules/*", "**/*.test.ts"],
```

Comment:
Is this WIP? Thinking that we don't want to get into a situation like we were in with reaction where fixing omitted test type-checking led to some late nights 😄

Reply:
Tests were being emitted when running the commands added in this PR - can you clarify why we want to emit tests?

Reply:
I removed tests intentionally because i didn't want them emitted in the published package, but you're right, it seems to cause issues locally.